cmd/ctr/commands/sandboxes/sandboxes.go

```diff
@@ -0,0 +1,178 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package sandboxes
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/cmd/ctr/commands"
+	"github.com/containerd/containerd/defaults"
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/oci"
+	"github.com/urfave/cli"
+)
+
+// Command is a set of subcommands to manage runtimes with sandbox support
+var Command = cli.Command{
+	Name:    "sandboxes",
+	Aliases: []string{"sandbox", "sb", "s"},
+	Usage:   "manage sandboxes",
+	Subcommands: cli.Commands{
+		runCommand,
+		listCommand,
+		removeCommand,
+	},
+}
+
+var runCommand = cli.Command{
+	Name:      "run",
+	Aliases:   []string{"create", "c", "r"},
+	Usage:     "run a new sandbox",
+	ArgsUsage: "[flags] <pod-config.json> <sandbox-id>",
```

Comment:
No pod-config.json, I think?

Reply:
This can be any spec recognized to a shim implementation.
These includes the container spec used when launching pause container: https://github.com/containerd/containerd/blob/887615c7d044af735f1d177e5afad595c976181c/pkg/cri/server/sandbox_run.go#L177